test/lazy-loading.html

```diff
@@ -925,6 +925,38 @@
               expect(comboBox.value).to.equal('item 12');
             });
 
+            it('should set value without auto-open-disabled even if case does not match', () => {
+              comboBox.autoOpenDisabled = false;
+              expect(comboBox.autoOpenDisabled).to.be.false;
+
+              comboBox._inputElementValue = 'ItEm 12';
+              comboBox.filter = 'item 12';
+              expect(comboBox.opened).to.be.false;
+              expect(comboBox.hasAttribute('focused')).to.be.false;
+              expect(comboBox.value).to.equal('item 12');
+            });
+
+            it('should set value with auto-open-disabled even if case does not match', () => {
+              comboBox.autoOpenDisabled = true;
+              expect(comboBox.autoOpenDisabled).to.be.true;
+
+              comboBox._inputElementValue = 'iTem 12';
+              comboBox.filter = 'item 12';
+              expect(comboBox.opened).to.be.false;
+              expect(comboBox.hasAttribute('focused')).to.be.false;
+              expect(comboBox.value).to.equal('item 12');
+            });
+
+            it('should set value with multiple matches that differ only in case', () => {
```

Comment:
If I understand it correctly, this test verifies that the first item is picked if there are multiple matches. I think the test name should reflect that.

Reply:
@tulioag What about this one?

Reply:
Done